src/go/rpk/pkg/cli/cmd/redpanda/tune/tune.go

```diff
@@ -41,7 +39,7 @@ type result struct {
 	errMsg    string
 }
 
-func NewTuneCommand(fs afero.Fs) *cobra.Command {
+func NewCommand(fs afero.Fs) *cobra.Command {
 	tunerParams := factory.TunerParams{}
 	var (
 		configFile        string
```

Comment:
Commenting here because I can't below: can the baseMsg below be shorter and inline after theShort field on line 57? Listing all the tuners in the short text seems verbose. I would expect only the long text to list all tuners, and not as a slice but instead in an ordered list with one tuner per line, and a description of the tuner after it. IMO this could largely replace the tune help command, actually.